libs/common/src/lib/create-store.ts

```diff
@@ -0,0 +1,133 @@
+import { Subscription } from 'rxjs';
+import {
+    AppState,
+    MetaReducer,
+    OperationType,
+    Reducer,
+    StoreConfig,
+    StoreExtension,
+} from './models';
+import { createReducerManager, ReducerManager } from './reducer-manager';
+import { createActionsOnQueue } from './actions-on-queue';
+import { miniRxError } from './mini-rx-error';
+import { sortExtensions } from './sort-extensions';
+import { createMiniRxActionType } from './create-mini-rx-action-type';
+import { ExtensionId } from './enums';
+
+export function createStore<SelectFnType>(appState: {
+    get: () => AppState;
+    set: (v: AppState) => void;
+    select: SelectFnType;
+}) {
+    let hasUndoExtension = false;
+    let actionSubscription: Subscription | undefined;
+
+    // REDUCER MANAGER
+    let reducerManager: ReducerManager | undefined;
+
+    function getReducerManager(): ReducerManager {
+        if (!reducerManager) {
+            reducerManager = createReducerManager();
+        }
+        return reducerManager;
+    }
+
+    // ACTIONS
+    const { actions$, dispatch } = createActionsOnQueue();
+
+    // Wire up the Redux Store: subscribe to the actions stream, calc next state for every action
+    // Called by `configureStore` and `addReducer`
```

Comment:
Not sure if the second line of this comment is relevant as I don't see `addReducer` anywhere in this file? And you probably want to avoid comments that are hard-coupled to the implementation.

Reply:
fixed the comment